AzureSpatialAnchorsSample/README.md

```diff
@@ -1,12 +1,39 @@
+---
+page_type: sample
+name: Azure Spatial Anchors Sample
+description: This sample project shows how to use Azure Spatial Anchors on the HoloLens 2 using Unity and OpenXR.
+languages:
+- csharp
+products:
+- windows-mixed-reality
+- hololens
+---
+
 # Azure Spatial Anchors Sample
 
+![License](https://img.shields.io/badge/license-MIT-green.svg)
+
+Supported Unity versions | Built with XR configuration
+:-----------------: | :----------------: |
+Unity 2020 or higher | OpenXR 0.9.2 |
```

Comment:
nit: Should we be explicit that this version belongs to this plugin?
```suggestion
Unity 2020 or higher | Mixed Reality OpenXR Plugin 0.9.2 |
```

There is a separate Unity OpenXR Plugin also in use, but it has a different version (1.1.1 for now). We shouldn't have to mention it; this plugin just has a dependency on it, but when stating versions, it might be nice to be explicit about which of the two plugins we're talking about.